deploy/kiali/kiali_cr.yaml

```diff
@@ -784,3 +784,9 @@ spec:
 # When empty, Kiali will try to guess this value from HTTP headers.
 #    ---
 #    web_schema: ""
+#
+# Define the history mode of kiali UI. This can only take
+# two possible values: either "browser" or "hash".
+# When empty, it will always be considered as browser
+#    ---
+#    web_history_mode: ""
```

Comment:
As per [coding standards](https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting), place this in alphabetical order within this group of settings - so `web_history_mode` should be placed before `web_root` and after `web_fqdn`.